### ISSUE TITLE
Galaxy Zoo: Talk struggling to render discussions with many embedded images

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#7481]
Galaxy Zoo has some discussions with a lot of embedded galaxy images, which are struggling to load. In Firefox, this manifests as the default alt text ("Example Alt Text") being shown in place of each image.

eg. https://www.zooniverse.org/projects/zookeeper/galaxy-zoo/talk/1269/1184766

This could be mitigated by adding `loading=lazy` to images in Talk comments, so that they are loaded when they scroll into view. At the moment, every image is requested when the page first loads, which seems to be overloading the remote image server.

- Original Talk discussion: https://www.zooniverse.org/talk/17/4034710
- Fixed by #7480.

**Comment by eatyourgreens** [comment]
Here's a video showing the missing images on Galaxy Zoo Talk in Firefox, where a workaround is to right-click on each image's alt text individually, and reload the missing image from its context menu.

https://github.com/user-attachments/assets/3a820afd-13ff-42b8-bf19-cfffe0b8caa1
